airbyte-cdk/python/setup.py

```diff
@@ -44,6 +44,7 @@
     packages=find_packages(exclude=("unit_tests",)),
     package_data={"airbyte_cdk": ["py.typed", "sources/declarative/declarative_component_schema.yaml"]},
     install_requires=[
+        "airbyte_protocol.models==0.1.0",
```

Comment:
How do we update the version of airbyte-protocol.models? Should this just be a manual commit when needed, or should we add some tooling/automation?

Reply:
We should add tooling/automation. If it's too much to scope in the time frame of the deliverable, however, we can leave that out for now and have it manual

Reply:
For tooling/automation purposes it's probably sufficient to simply just bump the patch version every time we merge. We can think of something fancier later when the need arises.